rfc/rfc010-multiple-operators.md

```diff
@@ -0,0 +1,64 @@
+# RFC010 Multiple operator service access
+
+|  |  |
+| :--- | :--- |
+| Nuts foundation | W.M. Slakhorst |
+| Request for Comments: 010 | Nedap |
+|  | January 2021 |
+
+## Multiple operator service access
+### Abstract
+
+This RFC describes an addition to [RFC003](rfc003-oauth2-auithorization.md) and [RFC006](rfc006-distributed-registry.md) on how to handle resource access when a legal party has multiple node operators. This is the case when a care organization uses multiple pieces of software from different SaaS providers. A SaaS provider can alter the care organization DID document so the DID document of the other provider is added as synonym. This is done by using the *also-known-as* relation.
```

Comment:
> This is the case when a care organization uses multiple pieces of software from different SaaS providers

'multiple pieces of software' can go here I think, just 'uses multiple SaaS providers'? Since they could be using different implementations of the Nuts spec, or they don't, it doesn't matter. What matters is that they're operated by different companies.